src/PhpDoc/Tag/DeprecatedTag.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\PhpDoc\Tag;
+
+class DeprecatedTag
```

Comment:
I think that this class does not have to exist, would be fine if ResolvedPhpDocBlock just returned true/false `isDeprecated`.

Reply:
The [PHPDoc specification](http://docs.phpdoc.org/references/phpdoc/tags/deprecated.html) defines that the `@deprecated` attribute can have a version and description. This would make it easier to implement those features.

Reply:
YAGNI 😊 and it complicates the current code